gcc/cp/contracts.cc

```diff
@@ -1738,7 +1975,16 @@ declare_handle_contract_violation ()
 	    return f;
 	}
 
-  tree id_exp = get_identifier ("experimental");
+  tree id_exp = NULL_TREE;
+
+  if (flag_contracts_nonattr)
+    {
+      id_exp = get_identifier ("contracts");
```

Comment:
We should consider putting this in experimental/contracts until it's finalised (but we can defer until we want to upstream)